Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -3602,160 +3603,160 @@ def main():
+        elif command == 'xdr-get-script-code':
             return_outputs(*get_script_code_command(client, args))
 
-        elif demisto.command() == 'xdr-action-status-get':
+        elif command == 'xdr-action-status-get':
             return_outputs(*action_status_get_command(client, args))
 
-        elif demisto.command() == 'get-modified-remote-data':
+        elif command == 'get-modified-remote-data':
             return_results(get_modified_remote_data_command(client, demisto.args()))
 
-        elif demisto.command() == 'xdr-run-script':
+        elif command == 'xdr-run-script':
             return_results(run_script_command(client, args))
 
-        elif demisto.command() == 'xdr-run-snippet-code-script':
+        elif command == 'xdr-run-snippet-code-script':
             return_results(run_snippet_code_script_command(client, args))
 
-        elif demisto.command() == 'xdr-get-script-execution-status':
+        elif command == 'xdr-get-script-execution-status':
             return_results(get_script_execution_status_command(client, args))
 
-        elif demisto.command() == 'xdr-get-script-execution-results':
+        elif command == 'xdr-get-script-execution-results':
             return_results(get_script_execution_results_command(client, args))
 
-        elif demisto.command() == 'xdr-get-script-execution-result-files':
+        elif command == 'xdr-get-script-execution-result-files':
             return_results(get_script_execution_result_files_command(client, args))
 
-        elif demisto.command() == 'xdr-get-cloud-original-alerts':
+        elif command == 'xdr-get-cloud-original-alerts':
             return_results(get_original_alerts_command(client, args))
 
-        elif demisto.command() == 'xdr-run-script-execute-commands':
+        elif command == 'xdr-run-script-execute-commands':
             return_results(run_script_execute_commands_command(client, args))
 
-        elif demisto.command() == 'xdr-run-script-delete-file':
+        elif command == 'xdr-run-script-delete-file':
             return_results(run_script_delete_file_command(client, args))
 
-        elif demisto.command() == 'xdr-run-script-file-exists':
+        elif command == 'xdr-run-script-file-exists':
             return_results(run_script_file_exists_command(client, args))
 
-        elif demisto.command() == 'xdr-run-script-kill-process':
+        elif command == 'xdr-run-script-kill-process':
             return_results(run_script_kill_process_command(client, args))
 
-        elif demisto.command() == 'endpoint':
+        elif command == 'endpoint':
             return_results(endpoint_command(client, args))
 
-        elif demisto.command() == 'xdr-get-endpoints-by-status':
+        elif command == 'xdr-get-endpoints-by-status':
             return_results(get_endpoints_by_status_command(client, args))
 
     except Exception as err:
-        if demisto.command() == 'fetch-incidents':
+        if command == 'fetch-incidents':
```

Comment:
you can remove the condition block, `return_error` is handling it properly now.

Reply:
Done